lib/rspec_profiling/collectors/statsd.rb

```diff
@@ -93,6 +93,11 @@ def format_file(path, max = 0)
         return str
       end
 
+      def format_date(date_str)
+        Date.strptime(date_str, '%a %b %d %H:%M:%S %Y')
+            .strftime('%Y-%m-%dT%H:%M:%S')
```

Comment:
Are we sure colons are OK in both Graphite & Grafana? Also depending on how it looks in Grafana, we co uld go more compact even `%Y%m%dT%H%M%S` :)

Reply:
If these formats are specific for Graphite, Graphana or something else, its strings may be constants with meaningful names like `GRAFANA_TIME_FORMAT`. If the choice of format to use is up on us, I'd prefer UNIX's one, nothing but milliseconds